pkg/maps/ipcache/ipcache.go

```diff
@@ -132,7 +132,7 @@ type RemoteEndpointInfo struct {
 }
 
 func (v *RemoteEndpointInfo) String() string {
-	return fmt.Sprintf("%d %d", v.SecurityIdentity, v.Key)
+	return fmt.Sprintf("%d %d %d", v.SecurityIdentity, v.Key, v.TunnelEndpoint)
```

Comment:
Isn't the `TunnelEndpoint` an IP address? Seems like formatting it like that would be more readable.

Reply:
@joestringer it's a `[4]byte`, I could use
```
ip := net.IPv4(v.TunnelEndpoint[0],v.TunnelEndpoint[1],v.TunnelEndpoint[2],v.TunnelEndpoint[3])
ip.String()
```
will that work for you?

Reply:
That sounds reasonable, so long as the byte-ordering is correct. I think it's in host byte-order anyway so this looks right to me.

Reply:
@joestringer I just noticed, is there any reason why `v.TunnelEndpoint` is not a `types.IPv4`?

Reply:
I don't think there's a particular reason. `uint32` presumably just matched the C side, but so long as the value is the same size and alignment we shouldn't need to care about this.